monitoring/alerts.yaml

```diff
@@ -0,0 +1,91 @@
+# Variables which should be replaced. Similar to grafana dashboards' __inputs section
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: service
+    type: constant
+    value: artesca-data-connector-s3api-metrics
+
+groups:
+- name: CloudServer
+  rules:
+
+  # As a platform admin I want to be alerted (warning) when the system errors are more than 3% of
+  # all the response codes
+  - alert: SystemErrorsWarning
+    expr: |
+      sum (http_requests_total{namespace="${namespace}", service="${service}", code=~"5.."})
+          / sum (http_requests_total{namespace="${namespace}", service="${service}"})
+        >= 0.03
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: "High ratio of system erors on {{ $labels.pod }}"
```

Comment:
Be careful if you apply a `sum(...)` you might loose you labels which makes that message incomplete

Reply:
I need to test this indeed, it is not clear to me how this would work... In "EXPLORE" i can choose to get either a series (with labels it seems) or a single value, and I don't know how it goes with alerts ; but it labels get removed, I don't see the point having these Go templates in the first place :-/

Anyway, I will look into this.

Reply:
if you `expr` is based on a metric with some labels you can get the labels in the description. if you expression results in a value with no labels (like in the case of a `sum(...)`) you loose all your labels. you have access to all the labels you can see if you put your expression in the prometheus UI or in grafana

Reply:
Ok, should work with `sum by(pod)`